### PR TITLE
Fix links to approved experiences on Hall Staff page

### PR DESCRIPTION
**Pull request by shaneallgeier**
Also shows Drafts for Hall Staff users now. And renames `Name` to `Title`.

### CODE REVIEW

File: exdb/tests/integration_tests.py [hunk 1]
```diff
@@ -541,6 +546,37 @@ def test_does_not_submit_invalid(self):
...
+    def test_hallstaff_allowed_to_edit_own_drafts(self):
+        e = self.create_experience('dr',
+                                   start=(now() + timedelta(days=1)),
+                                   end=(now() + timedelta(days=2)),
+                                   author=self.clients['hs'].user_object)
+        response = self.clients['hs'].get(reverse('edit', args=[e.pk]))
+        self.assertEqual(
+            response.status_code,
+            200,
+            'Hall Staff users SHOULD be allowed to their own "draft" experiences')
```
Comment (tom08) [inline exdb/tests/integration_tests.py]: Hall Staff users SHOULD be allowed to *edit* their own "draft" experiences

**Comment by tom08** [comment]
Hallstaff users still cannot see their own drafts.

File: exdb/views.py [hunk 2]
```diff
@@ -51,9 +51,11 @@ def get_hs_queryset(self):
         experience_approvals = ExperienceApproval.objects.filter(
             approver=self.request.user, experience__status='ad'
         )
+        next_approver_queue = Q(next_approver=self.request.user) & ~Q(status="dr")
```
Comment (tom08) [inline exdb/views.py]: Thinking about it, this should probably be:
`Q(next_approver=self.request.user) & Q(status='pe')`
Thoughts?

Comment (shaneallgeier) [inline exdb/views.py]: My original thought was to actually change the final `return` statement to:
```python
return experiences | self.get_ra_queryset()
```
Which should theoretically merge the two queries... but would take some testing to be sure everything worked properly. To answer your question: yeah, it should probably only show pending experiences where they're the next_approver.

Comment (tom08) [inline exdb/views.py]: That should work.  We do need to filter out only pending experiences in the next_approver queue in whatever the final query is.  Else hs can see drafts (or any other experience) where they are set as the next approver.
I can work on it or you, your choice.

Comment (shaneallgeier) [inline exdb/views.py]: I got dis

File: exdb/tests/integration_tests.py [hunk 3]
```diff
@@ -466,34 +468,51 @@ def test_number_of_experiences(self):
...
+                                              sub_type=self.create_sub_type(),
+                                              goal="Test Goal",
+                                              audience="b",
+                                              status="ad",
+                                              attendance=None,
+                                              next_approver=self.clients['hs'].user_object)[0]
+        ExperienceApproval.objects.create(experience=e1, approver=self.clients['hs'].user_object)
+        ExperienceApproval.objects.create(experience=e2, approver=self.clients['hs'].user_object)
         response = self.clients['hs'].get(reverse('home'))
         self.assertEqual(len(response.context["upcoming"]), 1, "There should be 1 experience in the next week")
```
Comment (aaaustin10) [inline exdb/tests/integration_tests.py]: It isn't immediately obvious what this test is checking for. Perhaps you should add a comment

**Comment by aaaustin10** [comment]
I approve.

**Pull request merged**